core/src/replay_stage.rs

```diff
@@ -730,7 +729,7 @@ impl ReplayStage {
         bank: &Bank,
         blocktree: &Blocktree,
         bank_progress: &mut ForkProgress,
-    ) -> Result<(Vec<Entry>, usize, u64, u64)> {
+    ) -> Result<(Vec<Entry>, usize)> {
```

Comment:
Time for a struct

Reply:
I think for return values like this where the there 
1) There are a small number of return values 
2) The values are logically disjoint
3) The function name is self-documenting. For instance, `get_slot_entries_with_shred_count`, will return the number of entries and the number of shreds that were deserialized, we wouldn't get more information by making this a struct.

Then it makes more sense to use a tuple over a struct. I changed the name of this function to be more clear about the return values/consistent with the blocktree API it calls.